### ISSUE TITLE
Bigdebuffs causing stuttering

### ISSUE THREAD

**Issue by HuMa69** [jordonwow/bigdebuffs#419]
I isolated every addon on my game and bigdebuffs its causing a lot of stuttering specially when there are so many players around me like 25 raids AV or Wintergrasp, can you check it pls? ty

**Comment by darkwowwow** [comment]
can confirm i have this problem ever since wotlk pre patch. particularly in 25men raids.

**Comment by tellinz** [comment]
yes this is a huge issue for me that I've also narrowed down to BD, I've had to disable BD anytime I join a 25+ raid

**Comment by Calzeru** [comment]
Can confirm. Disabling BigDebuffs fixes a lot of the stuttering in 25man raids in Wotlk classic.

**Comment by SCUR0** [comment]
Another confirm. It has been getting worse.
